setup.cfg

```diff
@@ -13,13 +13,13 @@ license_files =
   LICENSE.txt
 
 [options]
-python_requires = >= 3.7
+python_requires = >= 3.8
 setup_requires =
   setuptools >= 40.6
   pip >= 10
 include_package_data = true
 packages = find:
 install_requires =
-    Django>=3.2, <4.2
-    wagtail>=4.1, <5.0
+    Django>=3.2, <4.3
+    wagtail>=4.1, <5.2
```

Comment:
```suggestion
    wagtail>=4.1, <5.3
```

Reply:
I think it would be preferable to just ditch the upperbounds, but what do you think?
https://github.com/torchbox-forks/wagtail-makeup/pull/4/commits/08c68346d23c53240cdd48ea94aa1494a8ebe976